test/IntegrationTests/Helpers/MockZipkinCollector.cs

```diff
@@ -52,7 +52,10 @@ public MockZipkinCollector(ITestOutputHelper output, int port = 9080, int retrie
             try
             {
                 listener.Start();
-                listener.Prefixes.Add($"http://*:{port}/"); // Warning: Requires admin access
+
+                // For the mock collector to receive data from Windows containers we need to bind to all network interfaces.
+                var host = EnvironmentTools.IsWindowsAdministrator() ? "*" : "localhost";
```

Comment:
I would prefer adding an optional parameter `host` with `localhost` as default to the `MockZipkinCollector` instead of adding such conditional behavior in the test code. It would make the test code more readable, trustworthy, and predictable (add more buzzwords here 😄 ).

Take notice that using `localhost` for most of the tests is fine and less invasive.